app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,80 @@
+<nav id="navcontainer" class="navbar navbar-default navbar-fixed-top">
```

Comment:
`navbar-inverse` instead of `navbar-default` will give a dark background more in line with the existing Tracks look & feel.